src/wing/CCPACSWing.h

```diff
@@ -35,6 +35,7 @@
 #include "CTiglAbstractSegment.h"
 #include "CCPACSGuideCurve.h"
 #include "Cache.h"
+#include "CCPACSDuctAssembly.h"
```

Comment:
```suggestion
```

I don't think the `CCPACSDuctAssembly` is used anywhere in `CCPACSWing`. Development artifact?